test/metabase/db/schema_migrations_test.clj

```diff
@@ -2017,33 +2017,35 @@
-                                                        :attribute_remappings "{\"foo\", 1}"
-                                                        :permission_id        permission-id}]})
-            expected        {:group_id             1
-                             :table_id             table-id
-                             :attribute_remappings "{\"foo\", 1}"}]
-        (migrate!)
-        (is (=? expected (t2/select-one :sandboxes :table_id table-id)))
-        (migrate! :down 49)
-        (is (=? expected (t2/select-one :sandboxes :table_id table-id)))))))
+  ;; TODO (noahmoss): uncomment when fixed on mysql
+  (mt/test-drivers [:postgres :h2 #_:mysql]
+    (testing "Can we rollback to 49 when sandboxing is configured"
+      (impl/test-migrations ["v50.2024-01-10T03:27:29" "v50.2024-06-20T13:21:30"] [migrate!]
+        (let [db-id         (first (t2/insert-returning-pks! (t2/table-name Database) {:name       "DB"
+                                                                                       :engine     "h2"
+                                                                                       :created_at :%now
+                                                                                       :updated_at :%now
+                                                                                       :details    "{}"}))
+              table-id      (first (t2/insert-returning-pks! (t2/table-name Table) {:db_id      db-id
+                                                                                    :schema     "SchemaName"
+                                                                                    :name       "Table"
+                                                                                    :created_at :%now
+                                                                                    :updated_at :%now
+                                                                                    :active     true}))
+              permission-id (t2/insert-returning-pk! (t2/table-name :model/Permissions) {:object "/db/fake-permission/"
+                                                                                         :group_id 1})
+              _             (t2/query-one {:insert-into :sandboxes
+                                           :values      [{:group_id             1
+                                                          :table_id             table-id
+                                                          :attribute_remappings "{\"foo\", 1}"
+                                                          :permission_id        permission-id}]})
+              expected        {:group_id             1
+                               :table_id             table-id
+                               :attribute_remappings "{\"foo\", 1}"}]
+          (migrate!)
+          (is (=? expected (t2/select-one :sandboxes :table_id table-id)))
+          (migrate! :down 49)
```

Comment:
Not critical, but it should really just exclude MySQL on the rollback e.g.
`(when (not= driver/*driver* :mysql) (migrate! :down 49) ...etc)`

Reply:
In this case, the entire point of the test is to test the rollback, so I don't think it matters much

Reply:
True! Ignore my lazy comment